plugins/modules/source_control/gitlab/gitlab_group_members.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Zainab Alsaffar <Zainab.Alsaffar@mail.rit.edu>
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: gitlab_group_members
+short_description: Manage group members on GitLab Server
+description:
+    - This is a custom module built to manage the addition of members to a group on GitLab when state is defined as present. 
+    - And the removal of a user when state is defined as absent.
+version_added: '1.0'
+author: 'Zainab Alsaffar (@zanssa)'
+requirments:
+    - 'python >= 2.6'
+    - 'python-gitlab' python module <= 1.15.0
+    - administrator rights on the GitLab server
+options:
+    server_url:
+        description:
+            - URL of a GitLab server.
+        required: true
+        type: str
+    access_token:
+        description:
+            - A personal access token to authenticate with the GitLab API
+        required: true
+        type: str
+    gitlab_group:
+        description:
+            - The name of the GitLab group you want to add a member to.
+        required: true
+        type: str
+    gitlab_user:
+        description:
+            - The username of the member you want to add to the GitLab group.
+        required: true
+        type: str
+    access_level:
+        description:
+            - The access level for the user.
+            - Required if I(state=present), user state is set to present.
+        type: str
+        choices: ['guest','reporter','developer','maintainer','owner']
+    state:
+        description:
+            - State of the member in the group.
+            - On C(present), it will add a user to a GitLab group.
+            - On C(absent), will remove a user from a GitLab group.
+        choices: ['present', 'absent']
+        default: 'present'
+        type: str
+notes: supports_check_mode is allowed in this module 
```

Comment:
```suggestion
```
```suggestion
notes: Supports C(check_mode).
```
but i'd remove this note because imo this is extra information.